Directory.Build.props

```diff
@@ -36,7 +36,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)\src\codeAnalysis\GlobalSuppressions.cs" Link="GlobalSuppressions.cs" />
     <AdditionalFiles Include="$(MSBuildThisFileDirectory)\src\codeAnalysis\StyleCop.json" Link="StyleCop.json" />
 
-    <PackageReference Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="6.0.0">
+    <PackageReference Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="7.0.0-preview1.22464.1">
```

Comment:
Why a preview version here? Can this be a non preview version?

Reply:
At the time it still had only a preview version of this package. When left on `6.0.0` it will throw errors.

Reply:
Alright, thank you!
It's for analyzers only so it should be fine. Bringing this in ;)